bin/dotd

```diff
@@ -167,7 +167,7 @@ should_i "DTP" do
 
   should_i "create a new release of the code-dot-org repository on github" do
     wait_for "the production server to fetch the latest changes"
-    system "ssh -t gateway.code.org ssh -t production-daemon \"production/bin/create-release\""
+    system "ssh -t gateway.code.org ssh -t \"production-daemon \\\"cd production; ./bin/create-release\\\"\""
```

Comment:
![faint](https://cloud.githubusercontent.com/assets/1615761/22566572/60cf1a7a-e941-11e6-9cd1-824f43a7bd84.gif)

My eyes glaze over anytime I see nested escaped quotes.  Can we mix quote types to make this a little less painful?  Or even better, extract a helper `run_on('production-daemon', 'cd production; ./bin/create-release')` to hide the complexity?